### ISSUE TITLE
A11y - Extend Pa11y CI to cover more panel visualizations

### ISSUE THREAD

**Issue by axelavargas** [grafana/grafana#39331]
We are only collecting 13 pages on Pa11y CI, but we are missing a lot of panel visualizations, in order to improve our coverage recently we merged a [PR](https://github.com/grafana/grafana/pull/38113) that allow us to have a dashboard with all visualizations in the e2e grafana instance.

The idea of this issue is to extend `pa11y-ci` to test more panel visualizations.

Current pages covered by pa11y-ci:

1. Login
2. Home
3. Search
4. Panel Baur Gauge View
5. Panel Baur Gauge Edit
6. Alerting
7. Explore
8. Configuration: DataSources
9. Configuration: Users
10. Configuration: Teams
11. Configuration: Plugins
12. Configuration: Preferences
13. Configuration: API keys

Expected Visualizations to be covered, it should include view and edit mode: 

- [ ] Text Panel
- [ ] State timeline Panel
- [ ] Geomap
- [ ] Histogram
- [ ] Logs
- [ ] Dashboard List
- [ ] Panel List
- [ ] Alert List
- [ ] Heatmap
- [ ] Pie chart
- [ ] Table
- [ ] Anotations List
- [ ] Stat
- [ ] Timeseries
- [ ] BarChart
- [ ] Gauge
